### DISCUSSION TITLE
DPI

### DISCUSSION THREAD

**Discussion by LRostron** [photopea/photopea#5493] [Q&A]
My students are trying to save their work as PNG with a dpi of 300.  However when saved, the dpi converts to 96.  Please help

**Comment by photopea** [comment]
Hi, how exactly do you determine, that a PNG file has a DPI of 96?

I think that your tool can not detect DPI of a PNG file properly, so it shows 96 for every PNG file.

Can you find a PNG image, for which your tool shows a different DPI (not 96)? This question is not related to Photopea at all.

Also, if you send an image over social networs, the network could delete the DPI information.

**Reply by NeticDesigns** [reply]
I have a question about the DPI for Photopea generally, are you guys able to upscale said DPI? I am having somewhat of the same issue as the same guy, I make sure my DPI is set between 500-1k, and one of two times. The GFX will end pretty low quality, so I am just asking are you able to update DPI?

**Comment by photopea** [comment]
You can set any DPI in Image - Image Size. Everything in Photopea works the same as in Adobe Photoshop.

If it still does not do what you want, I think the problem is in your understanding of computer graphics. I recommend buying and reading a book about Adobe Photoshop.
